newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/utils.js

```diff
@@ -275,6 +277,10 @@ export const getExampleAndTemplateItemsForBuildSection = ({
       if (!privateGameTemplateActuallyAdded) {
         const exampleShortHeader =
           exampleShortHeadersWithThumbnails[exampleIndex];
+        if (!exampleShortHeader) {
+          hasReachedEnd = true;
```

Comment:
is it possible you reach the end at the end of the loop and this variable is not set to true?
Meaning that if you click on "see more", it will just change the button label?

Reply:
Good spot!